lib/decoder_impl.cc

```diff
@@ -770,7 +769,7 @@ namespace gr {
                                                        this->d_samples_per_symbol / this->d_corr_decim_factor,
                                                        &index_correction);
 
-                        if (c > 0.8f) {
+                        if (c > 0.9f) {
```

Comment:
This value is too high for the RTL-SDR. Either some filtering must be done or the correlation must be set to 0.8f